crates/neon/src/types_impl/extract/json.rs

```diff
@@ -0,0 +1,76 @@
+use crate::{
+    context::Context,
+    handle::Handle,
+    object::Object,
+    result::{JsResult, NeonResult},
+    types::{
+        extract::{private, TryFromJs},
+        JsFunction, JsObject, JsString, JsValue,
+    },
+};
+
+#[cfg(feature = "napi-6")]
+use crate::{handle::Root, thread::LocalKey};
+
+fn global_json_stringify<'cx, C>(cx: &mut C) -> JsResult<'cx, JsFunction>
+where
+    C: Context<'cx>,
+{
+    cx.global::<JsObject>("JSON")?.get(cx, "stringify")
+}
+
+#[cfg(not(feature = "napi-6"))]
```

Comment:
Pedantically, this isn't exactly equivalent behaviorally since someone could modify `JSON.stringify`, but I don't think it matters.